### PR TITLE
CSS as JS

### PR DESCRIPTION
**Pull request by jamesgorrie**
Adds some utility classes via the [`createGlobalStyle`](https://styled-components.com/docs/api#createglobalstyle) for showing and the filters on desktop (and beyond 🚀 ).

This should help reduce the jank on the form #6268

### CODE REVIEW

File: common/views/themes/default.ts [hunk 1]
```diff
@@ -185,4 +186,32 @@ const theme = {
...
+      display: block;
+    }
+    .none {
+      display: none;
+    }
+  `}
+  ${css`
+    ${Object.keys(themeValues.sizes).map(
+      size => css`
+        .${size}-none {
```
Comment (davidpmccormick) [inline common/views/themes/default.ts:201]: Could we go with `-show` and `-hide`?

Comment (jamesgorrie) [reply common/views/themes/default.ts:201]: The reason I have been avoiding that is `hide` is quite ambivalent, could be `visibility: hidden` or similar.

I have avoided using `-show` as you might want to use a different `display: ` value later, so this should help this scale.

I could prefix it with `{size}-display-show` or something?

Comment (davidpmccormick) [reply common/views/themes/default.ts:201]: yeah, I was mostly thinking about something to make it less ambiguous – `none` isn't obviously about `display`.

**Review by gestchild — APPROVED** [review]
Nice

**Review by davidpmccormick — APPROVED** [review]

**Pull request merged**